src/ast/value.rs

```diff
@@ -114,6 +115,10 @@ impl fmt::Display for Value {
                     write!(f, "{}", " }")
                 }
             }
+            Value::Array(values) => {
+                let collected: Vec<String> = values.iter().map(|v| format!("{}", v)).collect();
+                write!(f, "[{}]", collected.join(", "))
```

Comment:
There is a helper you should use instead , something delimited

Reply:
@lustefaniak I wanted to re-use `display_comma_separated`, but it is in module above `ast::mod` and It would be circular dependency.
Then I was thinking about copy that logic here, but saw in this file they are using same principle, but using `for` iteration. Check logic few lines above it is:
```
 for ObjectConstantKeyValue { key, value } in fields {
                        if first {
                            first = false;
                        } else {
                            write!(f, ", ")?;
                        }
                        write!(f, "'{}': {}", key, value)?;
                    }
```
So I kept it in way w/o helper how they have.
Is it ok?